ast/commandflag/flags.go

```diff
@@ -38,6 +38,7 @@ type RunOpts struct {
 	Secrets         []string `long:"secret" description:"Make available a secret"`
 	Mounts          []string `long:"mount" description:"Mount a file or directory"`
 	Network         string   `long:"network" description:"Network to use; currently network=none is only supported"`
+	RawOutput       bool     `long:"raw-output" description:"Do not prefix output with target. Print Raw"`
```

Comment:
we usually add a new feature flag in features.go when introducing a new command flag

Reply:
check out 2721c63ee6d923d6d801606913de47a90117c697 for an example of new feature that includes feature flagging.

Reply:
Done.